geode-core/src/main/java/org/apache/geode/distributed/ConfigurationProperties.java

```diff
@@ -1920,7 +1920,7 @@
    * clients to connect and store data in GemFire distributed system. see {@code GeodeRedisServer}
    * for other configuration options.
    * </p>
-   * <U>Default</U>: "0" disables GemFireMemcachedServer
+   * <U>Default</U>: "0" disables {@code GeodeRedisServer}
    * </p>
    * <U>Allowed values</U>: 0..65535
```

Comment:
Shouldn't the allowed values now be -1..65535?
Also seems like you should add a line to the description saying what -1 means.

Reply:
Yes. I updated the comment to mention `-1`